cypress/e2e/pokedex.cy.js

```diff
@@ -0,0 +1,124 @@
+  });
+
+  it("Clicks on pokemon card", () => {
+    cy.intercept("GET", blastoiseURL, { fixture: "blastoise" }).as("getBlastoise");
+    cy.intercept("GET", blastoiseSpeciesURL, { fixture: "blastoiseSpecies" }).as("getBlastoiseSpecies");
+
+    cy.get("[data-cy='Blastoise-card']").click();
+    cy.get("[data-cy='pokemon-card-modal']").should("be.visible");
+
+    cy.get("[data-cy='previous-evolution-name']").should("have.text", "Evolves from wartortle");
+    cy.get("[data-cy='pokemon-genus']").should("have.text", "Shellfish Pokémon");
+
+    cy.get("[data-cy='previous-evolution-sprite']").should("have.attr", "src", "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/8.png");
+    cy.get("[data-cy='pokemon-name']").should("have.text", "blastoise");
+    cy.get("[data-cy='pokemon-hp']").should("have.text", "79 HP");
+    cy.get("[data-cy='pokemon-main-type-icon']").should("have.attr", "src", "img/pokemon-types/icons/water-type-icon.png");
+
+    cy.get("[data-cy='pokemon-sprite']").should("have.attr", "src", "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/9.png");
+
+    cy.get("[data-cy='pokemon-main-type-logo']").should("have.attr", "src", "img/pokemon-types/logos/water-type.png");
+    cy.get("[data-cy='pokemon-secondary-type-logo']").should("have.attr", "src", "img/pokemon-types/logos/undefined-type.png");
+    cy.get("[data-cy='pokemon-height']").should("have.text", `Length: 5'25" `);
+    cy.get("[data-cy='pokemon-weight']").should("have.text", `Weight: 1.88 lbs`);
+
+    cy.get("[data-cy='first-skill-icon']").should("have.attr", "src", "img/pokemon-types/icons/water-type-icon.png");
+    cy.get("[data-cy='first-skill']").should("have.text", "torrent");
+    cy.get("[data-cy='second-skill-icon']").should("have.attr", "src", "img/pokemon-types/icons/water-type-icon.png");
+    cy.get("[data-cy='second-skill']").should("have.text", "rain-dish");
+
+    cy.get("[data-cy='pokemon-attack']").should("have.text", "ATTACK: 83");
+    cy.get("[data-cy='pokemon-defense']").should("have.text", "DEFENSE: 100");
+    cy.get("[data-cy='pokemon-speed']").should("have.text", "SPEED: 78");
+    cy.get("[data-cy='pokemon-sp-attack']").should("have.text", "SP ATK: 85");
+    cy.get("[data-cy='pokemon-sp-defense']").should("have.text", "SP DEF: 105");
+
+    cy.get("[data-cy='weakness-icon']").should("have.attr", "src", "img/pokemon-types/icons/electric-type-icon.png");
+    cy.get("[data-cy='resistance-icon']").should("have.attr", "src", "img/pokemon-types/icons/fire-type-icon.png");
+    cy.get("[data-cy='retreat-icon']").children().should("have.length", 3);
+
+    cy.get("[data-cy='pokemon-description']").should("have.text", "A brutal POKéMON\nwith pressurized\nwater jets on its shell. They are\nused for high\nspeed tackles.");
+
+    cy.get("[data-cy='pokemon-modal-close-button']").click();
+  });
+
+  it("Uses paginator", () => {
```

Comment:
Same here.

Reply:
Added it in here as well, thanks for the feedback !!